content/docs/guides/contribute/license-and-set-up.en.md

```diff
@@ -28,13 +28,11 @@ Most OSRD developers use Linux (incl. [WSL](https://learn.microsoft.com/en-us/wi
 
 ### Launch the application
 
-Thanks to `docker`, one can easily compile, configure, and run all services after making a change. One can also start only a subset of the services.
```

Comment:
Why did this line get removed? (and the French one is still here).

Reply:
I replace it with something more substancial